packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -85,26 +77,54 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
       initialsTextColor="white"
     />
   );
+
+  const meTextStyle = mergeStyles({ color: theme.palette.neutralTertiary }, styles?.me);
+  const contextualMenuStyle = mergeStyles({ background: theme.palette.neutralLighterAlt }, styles?.menu);
+  const menuButtonContainerStyle = mergeStyles({
+    root: {
+      selectors: {
+        '&:hover': { background: 'none' },
+        '&:active': { background: 'none' }
+      }
+    }
+  });
+
   return (
     <div
       ref={containerRef}
-      className={mergeStyles(participantItemContainerStyle(theme), styles?.root)}
-      onClick={showMenu}
+      className={mergeStyles(participantItemContainerStyle, styles?.root)}
+      onMouseEnter={() => setItemHovered(true)}
+      onMouseLeave={() => setItemHovered(false)}
     >
       {avatarToUse}
+      {me && <Stack className={meTextStyle}>{`(you)`}</Stack>}
       {onRenderIcon && (
-        <Stack className={mergeStyles(iconContainerStyle, styles?.iconContainer)}>{onRenderIcon(props)}</Stack>
-      )}
-      {menuItems && (
-        <ContextualMenu
-          items={menuItems}
-          hidden={menuHidden}
-          target={clickEvent ?? containerRef}
-          onItemClick={hideMenu}
-          onDismiss={hideMenu}
-          directionalHint={DirectionalHint.bottomLeftEdge}
-          className={mergeStyles(styles?.menu)}
-        />
+        <Stack horizontal={true} className={mergeStyles(iconContainerStyle, styles?.iconContainer)}>
+          {menuItems && menuItems.length > 0 && (itemHovered || !menuHidden) ? (
+            <div
```

Comment:
Is this div only the size of the icon? So you must click on the icon to show the menu? How does this work for mobile users, if the itemHovered if false, will this div still show up? Can the mobile users access the menu if they do not have the hover ability?